### PR TITLE
fix: start script

### PR DESCRIPTION
**Pull request by infloop**

### CODE REVIEW

**Review by Eddort — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "dev": "ts-node src/index.ts",
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js --verbose",
     "lint": "eslint --ext ts .",
-    "start": "node dist/index.js",
+    "start": "node dist/src/index.js",
```
Comment (avsetsin) [inline package.json:27]: why don't we exclude the src folder from the path during the build?

File: src/services/messages-processor/service.ts [hunk 2]
```diff
@@ -235,30 +238,33 @@ export const makeMessagesProcessor = ({
     eventsNumber: number
     messages: ExitMessage[]
   }) => {
-    logger.info('Job started')
+    logger.info('Job started', { OPERATOR_ID: config.OPERATOR_ID })
```
Comment (avsetsin) [inline src/services/messages-processor/service.ts:241]: maybe `OPERATOR_ID` -> `operatorId`

need to add module id as well

File: src/services/messages-processor/service.ts [hunk 3]
```diff
@@ -235,30 +238,33 @@ export const makeMessagesProcessor = ({
...
 
     const toBlock = await executionApi.latestBlockNumber()
     const fromBlock = toBlock - eventsNumber
-    logger.info(`Latest block is ${toBlock}`)
 
+    logger.info(`Latest block is ${toBlock}`)
     logger.info(
       `Fetching events for ${eventsNumber} last blocks (${fromBlock}-${toBlock})`
     )
```
Comment (avsetsin) [inline src/services/messages-processor/service.ts:249]:
```suggestion
    logger.info(`Fetching request events from the Exit Bus`, { eventsNumber, fromBlock, toBlock } )
```

File: package.json (hunk 1 above)
Comment (kolyasapphire) [reply package.json:27]: Yep, I'm in favour of `rootDir`

**Comment by kolyasapphire** [comment]
Moved the suggestions to #36

File: package.json (hunk 1 above)
Comment (kolyasapphire) [reply package.json:27]: Decided to keep this in order to have both the app and encryptor script available after build in the docker image

File: src/services/messages-processor/service.ts (hunk 3 above)
Comment (kolyasapphire) [reply src/services/messages-processor/service.ts]: Moved this comment to the original PR #36

File: src/services/messages-processor/service.ts (hunk 2 above)
Comment (kolyasapphire) [reply src/services/messages-processor/service.ts]: Moved this comment to the original PR #36

**Pull request merged**